features/credit_card_url.feature

```diff
@@ -30,3 +30,12 @@ Feature: Credit Card Form URL
 
     When I visit my profile page
     Then the new value should be stored as my Credit Card Form URL
+
+  Scenario: Vendor cannot save credit card URL with invalid format
```

Comment:
I just wanted to take what was on the card, but you are right that I have also added some unit tests for this already.